docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -0,0 +1,21 @@
+# Component technologies
+
+As a complete microservices platform, Lagom assembles a collection of technologies and adds value on top of them. Some of the libraries, tools, and servers that Lagom uses and supports were developed at [Lightbend](https://lightbend.com), others are third-party and open-source. As you develop with Lagom, you can also take advantage of these technologies:
+
+* Akka --- Lagom [[Persistence|PersistentEntity]], [[Publish-Subscribe|PubSub]], and [[Cluster|Cluster]] are implemented on top of [Akka](http://akka.io/), Lightbend's toolkit for building concurrent, distributed, and resilient message-driven applications. This is an implementation detail that does not directly concern you when developing simple microservices. However, advanced users can call some Akka APIs directly. Specifically:
```

Comment:
I still think it doesn't read quite right. The nested bullets aren't exactly APIs that you call directly. In the original document, the clustering/SBR, and Akka Streams parts were in separate sub-headings, not linked to the statement about calling Akka APIs directly.

Maybe the best thing to do would be to remove the nested bullets, as these points are all covered elsewhere in the documentation.

What's the reasoning for removing the link to the "Advanced Topic: Integrating with Akka" page?